examples/seq2seq/seq2seq_trainer.py

```diff
@@ -217,7 +217,12 @@ def prediction_step(
         }
 
         if self.args.predict_with_generate and not self.args.prediction_loss_only:
-            generated_tokens = self.model.generate(
+            if self.args.deepspeed:
+                model = self.model.module
```

Comment:
I thought we had `self.model` containing the inner model now?

Reply:
nice catch! this is from the first days when it wasn't and in a different file. will fix.